### PR TITLE
Add + icon next to available questions in Question Bank.

### LINKED ISSUE #1013: Add "+" button next to questions in the form builder
This will make it more obvious to the admin that they have to click it to add the question to the block. Right now there's no affordance for that.

### PR DESCRIPTION
**Pull request by shanemc-goog**
### Description
Add + icon next to questions in the Question Bank to give affordance that they can be selected to add them to the program..

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #1013

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/components/Icons.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
           + " 100-2H7zm2 1a1 1 0 011-1h.01a1 1 0 110 2H10a1 1 0 01-1-1zm4-4a1 1 0 100 2h.01a1 1 0"
           + " 100-2H13zM9 9a1 1 0 011-1h.01a1 1 0 110 2H10a1 1 0 01-1-1zM7 8a1 1 0 000 2h.01a1 1 0"
           + " 000-2H7z";
+  public static final String PLUS_SVG_PATH =
+      "M10 5a1 1 0 011 1v3h3a1 1 0 110 2h-3v3a1 1 0 11-2 0v-3H6a1 1 0 110-2h3V6a1 1 0 011-1z";
```
Comment (ettengererin) [inline universal-application-tool-0.0.1/app/views/components/Icons.java:43]: For my own learning -- how did you find/create this constant?

**Review by ettengererin — COMMENTED** [review]
This is great, nice work! You mentioned you want to upload screenshots, so I'd like to see those before I approve.

**Comment by shanemc-goog** [comment]
image

File: universal-application-tool-0.0.1/app/views/components/Icons.java (hunk 1 above)
Comment (shanemc-goog) [reply universal-application-tool-0.0.1/app/views/components/Icons.java:43]: I looked into the slack history and saw someone else ask :D The recommendation was to use https://heroicons.com/ or https://ikonate.com.  This is pulled from heroicons which the question type icons look to be from too.

**Review by ettengererin — APPROVED** [review]
Nice, great update! I've been confused about how to add questions as I'm testing as an admin, and this will really help.

**Pull request merged**